tests/test_casereader.py

```diff
@@ -15,9 +19,9 @@ def call_casereader(
     if expected is not None:
         assert reader.precision() == expected["precision"]
         assert reader.num_dimensions() == expected["num_dimensions"]
-        assert {p.name: p.value for p in reader.input_parameters()} == expected[
-            "input_parameters"
-        ]
+        assert {
+            p.name: (p.number, p.units) for p in reader.input_parameters()
```

Comment:
@tejalprabhu01 one needs to access `number` and `units` attributes of the input parameter object. `value` is retaining its original meaning (it is used as such in the pyfluent-parametric library).